### ISSUE TITLE
Add alt text to Indian payor handbook.

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1663]

**Comment by cthomasONRR** [comment]
The following content blocks will not save alt text:

- [ ] 2.5.1

**Comment by mcharg** [comment]
It's not saving alt text for me in those content blocks either.

**Comment by mcharg** [comment]
@cthomasONRR Looks good.  Didn't find any with missing alt text.
